### PR TITLE
change config description to avoid ambiguity

### PR DESCRIPTION
**Pull request by hawkingrei**
### What is changed:

change configure's description

### CODE REVIEW

File: get-started/setup-an-ide.md [hunk 1]
```diff
@@ -14,12 +14,11 @@ You can use [GoLand](https://www.jetbrains.com/go/) to easily run or debug TiDB
 
 ![tidb in goland](https://user-images.githubusercontent.com/30543181/118812264-7fb9cc00-b8e0-11eb-8825-223afa93b2f0.png)
 
-**Step 4**. Add these three config files to `<tidb-dir>/.idea/runConfigurations/` directory by executing following commands or adding them manually.
+**Step 4**. you need to replace ```{{PD_ADDRESS}}``` with your pd address (such as 127.0.0.1:2379) for following scipts. ```cd <tidb-dir>``` and Executing it to adding these three config files to `<tidb-dir>/.idea/runConfigurations/` directory.
```
Comment (tisonkun) [inline get-started/setup-an-ide.md:17]:
```suggestion
**Step 4**. you need to replace `{{PD_ADDRESS}}` with your pd address (such as 127.0.0.1:2379) for following scipts. `cd <tidb-dir>` and Executing it to adding these three config files to `<tidb-dir>/.idea/runConfigurations/` directory. 
```

Comment (tisonkun) [reply get-started/setup-an-ide.md:17]: It is a bit duplicated we mention two times about the working directory.

**Comment by tisonkun** [comment]
cc @ichn-hu

File: get-started/setup-an-ide.md (hunk 1 above)
Comment (hawkingrei) [reply get-started/setup-an-ide.md]: Delete the last one.

**Comment by ichn-hu** [comment]
why changing it? For first-time contributor, they would not know anything about PD, and the use of the default port seems totally valid.

**Comment by hawkingrei** [comment]
> why changing it? For first-time contributor, they would not know anything about PD, and the use of the default port seems totally valid.

OK,I understand. but we don't use ```pd0:2379``` to express the pd address. Contributor would not know angthing about ```pd0```. so I replace it with ```{{ PD ADDRESS}}``` and add default address to explain this magic character.

File: get-started/setup-an-ide.md [hunk 2]
```diff
@@ -33,12 +32,12 @@ $ cat <<EOF > unistore_4000.xml
...
 EOF
 
-$ cat <<EOF > playground_attach_4001.xml
+cat <<EOF > playground_attach_4001.xml
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="playground attach 4001" type="GoApplicationRunConfiguration" factoryName="Go Application">
     <module name="tidb" />
     <working_directory value="$PROJECT_DIR$" />
-    <parameters value="--path=pd0:2379 --store=tikv --status=10081 -P 4001 " />
+    <parameters value="--path={{PD_ADDRESS}} --store=tikv --status=10081 -P 4001 " />
```
Comment (ichn-hu) [inline get-started/setup-an-ide.md:40]:
```suggestion
    <parameters value="--path=127.0.0.1:2379 --store=tikv --status=10081 -P 4001 " />
```

**Comment by ichn-hu** [comment]
I prefer the suggestion above, there is no reason why we want the user to change it themselvfs.

**Comment by hawkingrei** [comment]
> I prefer the suggestion above, there is no reason why we want the user to change it themselvfs.

Modified as you suggested

**Review by ichn-hu — APPROVED** [review]

File: get-started/setup-an-ide.md [hunk 3]
```diff
@@ -14,12 +14,11 @@ You can use [GoLand](https://www.jetbrains.com/go/) to easily run or debug TiDB
 
 ![tidb in goland](https://user-images.githubusercontent.com/30543181/118812264-7fb9cc00-b8e0-11eb-8825-223afa93b2f0.png)
 
-**Step 4**. Add these three config files to `<tidb-dir>/.idea/runConfigurations/` directory by executing following commands or adding them manually.
+**Step 4**. ```cd <tidb-dir>``` and Executing it to adding these three config files to `./.idea/runConfigurations/` directory.
```
Comment (tisonkun) [inline get-started/setup-an-ide.md:17]:
```suggestion
**Step 4**. `cd <tidb-dir>` and Executing it to adding these three config files to `./.idea/runConfigurations/` directory. 
```

**Review by tisonkun — APPROVED** [review]
LGTM

File: get-started/setup-an-ide.md [hunk 4]
```diff
@@ -14,12 +14,11 @@ You can use [GoLand](https://www.jetbrains.com/go/) to easily run or debug TiDB
 
 ![tidb in goland](https://user-images.githubusercontent.com/30543181/118812264-7fb9cc00-b8e0-11eb-8825-223afa93b2f0.png)
 
-**Step 4**. Add these three config files to `<tidb-dir>/.idea/runConfigurations/` directory by executing following commands or adding them manually.
+**Step 4**. `cd <tidb-dir>` and Executing it to adding these three config files to `./.idea/runConfigurations/` directory.
```
Comment (tisonkun) [inline get-started/setup-an-ide.md:17]:
```suggestion
**Step 4**. `cd <tidb-dir>` and execute following commands to add these three config files to `./.idea/runConfigurations/` directory. 
```

File: get-started/setup-an-ide.md [hunk 5]
```diff
@@ -64,13 +63,15 @@ $ cat <<EOF > unit_test.xml
   </configuration>
 </component>
 EOF
+```
 
+After executing script, you can check whether to have three config in the directory. 
+```bash
```
Comment (tisonkun) [inline get-started/setup-an-ide.md:69]:
```suggestion
After executing script, you can check whether to have three config in the directory. 

```bash
```

**Comment by tisonkun** [comment]
@all-contributors please add @hawkingrei for content

**Pull request merged**